gstcefbin.cc

```diff
@@ -13,7 +13,7 @@ gst_cef_bin_uri_get_type (GType type)
 static const gchar *const *
 gst_cef_bin_get_protocols (GType type)
 {
-  static const gchar *protocols[] = { "web", "cef", NULL };
```

Comment:
I would like to keep the `cef` option too, when users want more control over the web backend :)

Reply:
can't the play with GST_PLUGIN_FEATURE_RANK?

Reply:
ah yes, I remember now, when I exposed this I didn't know there was an env var for ranks, only discovered that recently :) nvm me then.